packages/react-core/src/components/Alert/Alert.tsx

```diff
@@ -87,6 +87,14 @@ export interface AlertProps extends Omit<React.HTMLProps<HTMLDivElement>, 'actio
   ouiaSafe?: boolean;
 }
 
+const variantStyle = {
+  default: styles.modifiers.custom,
```

Comment:
It doesn't look like anything was decided on in the linked issue. In core "default" may conflict with keywords but a "default" string value for react shouldn't conflict I don't think, that's why I thought we were keeping "default" in react.

Reply:
Yes.  I believe some discussion happend in the leads meeting.

Reply:
@kmcfaul This was a long and difficult discussion, but we landed on "custom" mainly because it represents the usage we intended for this styling option. In the case of Alerts, for example, the ask was that we support the ability to create a custom alert with a consumer specified icon and color, so the word "default" was really not accurate as there is no such thing as a default alert. But it's really something more like a user defined alert, so "custom" seemed like the best term. Does that help?

Reply:
Updating to `custom` to keep the PR moving and match core.

Reply:
@kmcfaul agree, though to touch on what @mcarrano said, I think the custom variant is more of a wildcard/"other" variant (can be used for anything that isn't one of the existing success, error, etc statuses) with a pre-defined set of colors PF offers for whatever that's used for, that is unique to that variant. And anywhere we offer a status message/thing (alert, modal, popover, icon, icon, notification drawer), this other variant is available and there is consistency with the presentation. And if a user wants it to be purple because their brand is purple, they can update the `--pf-global--custom-color--[whatever]` variable(s) and it will update any component that offers the custom variant. I think that's probably the main value in offering an extra status variant.

From there, if they want a unique alert (or modal, popover, etc) variant specifically, like you said, they could just leave off the variant and use their own icon and CSS for the colors.

Reply:
Ah ok, having `pf-m-custom` tied to the `--pf-global--custom-color--{x}` makes it make a bit more sense to me now. I'm still worried there will be confusion though because the word doesn't indicate the user to anything.

Reply:
Had to update the docs a bit to reference the new enum value so made a note there